app/components/windows/OneOffWindow.vue

```diff
@@ -1,12 +1,12 @@
 <template>
 <div style="height: 100%">
   <title-bar :title="options.title" class="child-window-titlebar" v-if="!options.isFullScreen" />
-  <div class="blank-slate" v-if="!options.hideBlankSlate">
+  <div v-bind:class="['blank-slate', {'no-titlebar': options.isFullScreen}]" v-if="!options.hideBlankSlate">
     <div class="spinner-spacer" />
     <i class="icon-spinner icon-spin modal-layout-spinner"/>
     <div class="spinner-spacer" />
   </div>
-  <div class="child-window-content">
+  <div v-bind:class="['child-window-content', {'no-titlebar': options.isFullScreen}]">
```

Comment:
same https://github.com/n-air-app/n-air-app/pull/480/files#r540775981
```suggestion
  <div :class="['child-window-content', {'no-titlebar': options.isFullScreen}]">
```